tools/image_processing/imagej2/imagej2_macros.xml

```diff
@@ -1,20 +1,12 @@
 <macros>
-    <token name="@TOOL_VERSION@">20170530</token>
+    <token name="@TOOL_VERSION@">20231211</token>
```

Comment:
```suggestion
    <token name="@TOOL_VERSION@">20240614</token>
```

Latest version. Its not a must and will probably require more work. Just wanted to point out that we created a new package. We can do this as a followup if you like.